domains/authorization/Authorization.API/API/Authorization/[Features]/GrantConsentToClientCommand.cs

```diff
@@ -13,29 +13,29 @@
 
 namespace API.Authorization._Features_;
 
-public class GrantConsentCommandHandler(
+public class GrantConsentToClientCommandHandler(
     IClientRepository clientRepository,
     IOrganizationRepository organizationRepository,
     IUserRepository userRepository,
     IOrganizationConsentRepository organizationConsentRepository,
     IUnitOfWork unitOfWork)
-    : IRequestHandler<GrantConsentCommand>
+    : IRequestHandler<GrantConsentToClientCommand>
 {
-    public async Task Handle(GrantConsentCommand command, CancellationToken cancellationToken)
+    public async Task Handle(GrantConsentToClientCommand toClientCommand, CancellationToken cancellationToken)
```

Comment:
kan mærke det klør.

Det klør fordi den er navngivet toClientCommand. Er det grundet noget bestemt?

Reply:
Done, renamed to simply _command_, the context hints at what command it is.